readthedocs/settings/base.py

```diff
@@ -266,6 +266,9 @@ def USE_PROMOS(self):  # noqa
         },
     }
 
+    # Sentry
+    SENTRY_CELERY_IGNORE_EXPECTED = True
```

Comment:
This setting comes from https://docs.sentry.io/clients/python/integrations/django/#additional-settings (just linking for reference)

Reply:
Thank you.
I should have added this in the description.